hpy/devel/include/universal/meth.h

```diff
@@ -35,15 +35,18 @@ typedef struct {
 
 #define HPy_DECL_METH_O(NAME) HPy_DECL_METH_NOARGS(NAME)
 #define HPy_DECL_METH_VARARGS(NAME) HPy_DECL_METH_NOARGS(NAME)
+#define HPy_DECL_METH_KEYWORDS(NAME) HPy_DECL_METH_NOARGS(NAME)
 
 
 #define HPy_DEF_METH_NOARGS(fnname)                                            \
     static HPy fnname##_impl(HPyContext ctx, HPy self);                        \
     static struct _object *                                                    \
-    fnname##_trampoline(struct _object *self, struct _object *noargs)          \
+    fnname##_trampoline(struct _object *self, struct _object *noargs,          \
+                        struct _object *nokw)                                  \
```

Comment:
...and this is wrong, because CPython will call it using its own `PyCFunction` signature

Reply:
Fixed!